sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -761,13 +852,14 @@ class HistGradientBoostingClassifier(BaseHistGradientBoosting,
         The number of tree that are built at each iteration. This is equal to 1
         for binary classification, and to ``n_classes`` for multiclass
         classification.
-    train_score_ : ndarray, shape (max_iter + 1,)
+    train_score_ : ndarray, shape (n_iter_ + 1,)
```

Comment:
I don't understand why our rendered html generates a (void) reference for this.

@adrinjalali 	could this be linked to the recent changes made to sphinx?

Reply:
I really doubt that, the change was only not to raise a warning if a reference is not found, nothing else is changed AFAIK.